setup.py

```diff
@@ -55,7 +55,7 @@ def desc():
         "Flask-OpenID>=1.2.5, <2",
         # SQLAlchemy 1.4.0 breaks flask-sqlalchemy and sqlalchemy-utils
         "SQLAlchemy<1.4.0",
-        "Flask-SQLAlchemy>=2.4, <3",
+        "Flask-SQLAlchemy>=2.4, <5",
```

Comment:
Should we unpin `"SQLAlchemy<1.4.0"` - L57 in this PR itself?

Reply:
for sure, thats a mistake! thks